src/neki.jl

```diff
@@ -1,28 +1,28 @@
 """
-    neki_iter(prob::SolusProblem, θs, fθs)
+    neki_iter(prob::CESProblem, θs, fθs)
 
-Peform an iteration of NEKI, returning a new set of proposed `θs`.
+Perform an iteration of NEKI, returning a new set of proposed `θs`.
 
 See eqs. (5.4a-b) from https://arxiv.org/abs/1903.08866.
 """
-function neki_iter(prob::SolusProblem, θs, fθs)
+function neki_iter(prob::CESProblem, θs, fθs)
     covθ = cov(θs)
     meanθ = mean(θs)
-    
+
     m = mean(fθs)
 
     J = length(θs)
     CG = [dot(fθk - m, fθj - prob.obs, prob.space)/J for fθj in fθs, fθk in fθs]
 
     Δt = 1.0 / (norm(CG) + 1e-8)
```

Comment:
```suggestion
    Δt = 1.0 / (norm(CG) +eps)
```

Reply:
Since this may change the results, I'm going to leave this for a later PR and open an issue.